mobile-app/cypress/e2e/functional/wallet/loans/create_vault.spec.ts

```diff
@@ -44,17 +75,58 @@ context("Wallet - Loans - Create vault", () => {
         });
     });
   });
+});
 
-  it("should allow to submit", () => {
-    cy.getByTestID("bottom_tab_loans").click();
-    cy.getByTestID("create_vault_header_button").click();
-    cy.getByTestID("loan_scheme_option_0").click();
+context("Wallet - Loan - Create vault summary", () => {
+  beforeEach(() => {
+    cy.createEmptyWallet(true);
+    cy.sendDFItoWallet().sendDFITokentoWallet().wait(6000);
+  });
+
+  it("should display convert message", () => {
+    setupWalletForConversion();
+    navigateToVault("0");
+    cy.getByTestID("create_vault_summary").should("not.exist");
+    cy.getByTestID("action_message").should(
+      "have.text",
+      "By continuing, the required amount of DFI will be converted"
+    );
+    cy.getByTestID("create_vault_submit_button").should(
+      "have.text",
+      "Continue"
+    );
     cy.getByTestID("create_vault_submit_button").should(
       "not.have.attr",
       "disabled"
     );
-    cy.getByTestID("create_vault_submit_button").click();
-    cy.url().should("include", "ConfirmCreateVaultScreen");
+    cy.getByTestID("create_vault_submit_button").click().wait(3000);
+    cy.getByTestID("txn_authorization_title").should(
+      "have.text",
+      `Convert 2.00003301 DFI to UTXO`
```

Comment:
maybe use checkValueWithinRange(, , 0.0001) or something similar to reduce flakiness
<img width="530" alt="image" src="https://user-images.githubusercontent.com/13292662/191643616-448fb681-552a-4c31-9f5a-9d5649ebc636.png">

Reply:
Aside from this, everything LGTM! 🎉

Reply:
Done, update to use checkValueWithinRange